instructor-info/instructor-info.md

```diff
@@ -0,0 +1,68 @@
+---
+Front End Instructor Contact Information
+---
+
+# Instructors
+
+* Steve Kinney - Director
+  * Slack - @steve
+  * Github - [stevekinney](https://github.com/stevekinney)
+  * For Pairing - Send Slack DMs to schedule a time
+
+
+* Louisa Barrett
+  * Slack - @louisa
+  * Github - [LouisaBarrett](https://github.com/LouisaBarrett)
+  * For Pairing - Send Slack DMs to schedule a time
+
+
+* Romeeka Gayhart
+  * Slack - @meeka
+  * Github - [rrgayhart](https://github.com/rrgayhart)
+  * For Pairing - Send Slack DM or use her [YouCanBookMe](https://meeka.youcanbook.me/)
+
+
+
+* Jhun De Andres
+  * Slack - @jhunbug
+  * Github - [joshuajhun](https://github.com/joshuajhun)
+  * For pairing - Send Slack DMs to schedule a time
+
+
+* Taylor Moore
+  * Slack - @tmoore
+  * Github - [Tman22](https://github.com/Tman22)
+  * For Pairing - Send Slack DMs to schedule a time
+
+
+
+* Brittany Storoz
+  * Slack - @brittany
+  * Github - [brittanystoroz](https://github.com/brittanystoroz)
+  * Twitter - @brittanystoroz
+  * For pairing - Send Slack DMs to schedule a time
+
+
+* Brenna Martenson
+  * Slack - @brenna
+  * Github - [martensonbj](https://github.com/martensonbj)
+  * For pairing - Send Slack DMs to schedule a time
+
+
+* Bree Thomas
+  * Slack - @breethomas
+  * Github - [breethomas](https://github.com/breethomas)
+  * For pairing - Send Slack DMs to schedule a time
+
+
+* Robbie Jaeger
+  * Slack - @robbiejaeger
+  * Github - [robbiejaeger](https://github.com/robbiejaeger)
+  * For Pairing - Send Slack DMs to schedule a time
+
+### Student Support
```

Comment:
This looks great! 

I wonder if we should actually put Student Support at the top of the document - just because that would be the person you would be most likely to need to access.

Also, not sure if this is cluttered - but what about linking the pairing schedule with 1608 in here?